pkg/kubelet/types/types_test.go

```diff
@@ -136,3 +138,48 @@ func TestSortInitContainerStatuses(t *testing.T) {
 		}
 	}
 }
+
+func TestSortStatusesOfInitContainers(t *testing.T) {
+	pod := v1.Pod{
+		Spec: v1.PodSpec{},
+	}
+	var tests = []struct {
+		containers     []v1.Container
+		statusMap      map[string]*v1.ContainerStatus
+		expectStatuses []v1.ContainerStatus
+	}{
+		{
+			containers:     []v1.Container{{Name: "first"}, {Name: "second"}, {Name: "third"}, {Name: "fourth"}},
+			expectStatuses: []v1.ContainerStatus{{Name: "first"}, {Name: "second"}, {Name: "third"}, {Name: "fourth"}},
+			statusMap:      map[string]*v1.ContainerStatus{"first": {Name: "first"}, "second": {Name: "second"}, "third": {Name: "third"}, "fourth": {Name: "fourth"}},
+		},
+		{
+			containers:     []v1.Container{{Name: "first"}, {Name: "second"}, {Name: "third"}, {Name: "fourth"}},
+			expectStatuses: []v1.ContainerStatus{{Name: "first"}, {Name: "second"}, {Name: "third"}, {Name: "fourth"}},
+			statusMap:      map[string]*v1.ContainerStatus{"second": {Name: "second"}, "third": {Name: "third"}, "first": {Name: "first"}, "fourth": {Name: "fourth"}},
+		},
+		{
+			containers:     []v1.Container{{Name: "first"}, {Name: "second"}, {Name: "third"}, {Name: "fourth"}},
+			expectStatuses: []v1.ContainerStatus{{Name: "first"}, {Name: "third"}, {Name: "fourth"}},
+			statusMap:      map[string]*v1.ContainerStatus{"third": {Name: "third"}, "first": {Name: "first"}, "fourth": {Name: "fourth"}},
+		},
+		{
+			containers:     []v1.Container{{Name: "first"}, {Name: "second"}, {Name: "third"}, {Name: "fourth"}},
+			expectStatuses: []v1.ContainerStatus{{Name: "first"}, {Name: "third"}, {Name: "fourth"}},
+			statusMap:      map[string]*v1.ContainerStatus{"first": {Name: "first"}, "third": {Name: "third"}, "fourth": {Name: "fourth"}},
+		},
+	}
+	for _, data := range tests {
+		pod.Spec.InitContainers = data.containers
+		result := SortStatusesOfInitContainers(&pod, data.statusMap)
+		require.Equal(t, result, data.expectStatuses, "Unexpected result from SortStatusesOfInitContainers: %v", result)
+	}
+}
+
+func TestGet(t *testing.T) {
+	timestamp := NewTimestamp()
+	timestampString := timestamp.GetString()
+	timeParse, _ := time.Parse(RFC3339NanoLenient, timestampString)
+	timeResult := ConvertToTimestamp(timestampString).Get()
+	assert.Equal(t, timeResult, timeParse)
+}
```

Comment:
```suggestion
}
func TestNewTimestamp(t *testing.T) {
	timeStart := time.Now()
	timestamp := NewTimestamp()
	timeEnd := time.Now()
	assert.WithinDuration(t, timestamp.Get(), timeStart, timeEnd.Sub(timeStart))
}
```
I think you needed to use NewTimestamp() for 100% of coverage.
I think your test code alone would be fine, but
I would suggest a test code like the above for NewTimestamp(),

Reply:
My suggestion follows the comments in the source code below.

```
// NewTimestamp returns a Timestamp object using the current time.
func NewTimestamp() *Timestamp {
	return &Timestamp{time.Now()}
}
```

Reply:
Thank you for your suggestion, which I also agree with and adopt.